web/src/routes/+page.svelte

```diff
@@ -2,7 +2,7 @@
   import Checkbox from '$lib/elements/ui/checkbox/checkbox.svelte';
 </script>
 
-<h1>Welcome to SvelteKit</h1>
+<h1>Welcome to Svelte</h1>
```

Comment:
Technically SvelteKit is more correct :)

Reply:
Ahaha, the change was inintentional. It was made testing the HRM :)